### ISSUE TITLE
Torturer Garb Base does not import into POB

### ISSUE THREAD

**Issue by Bucketless1** [PathOfBuildingCommunity/PathOfBuilding#7992]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate open **and closed** issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues?q=is%3Aissue)

### What platform are you running Path of Building on?

Windows

### How is Path of Building expected to behave?

When I import character into PoB, it should import all items, tree and all skill gems.

### How does Path of Building behave?

Importing results in new base (Torturer Garb) not being imported, skill gems are not imported too
![image](https://github.com/user-attachments/assets/7bfc3be4-2368-4127-9f9f-997b2a7af890)
![image](https://github.com/user-attachments/assets/8bf24ba1-ea94-4750-b7e9-db302bfb0a94)

### How to reproduce the issue

1. Equip "Torturer Garb" base
2.  Go to import/export build tab.
3. Select Settlers league, select character.
4.  Uncheck "Delete jewels"; "Delete skills"; "Delete equipment".
5. Click Import passive tree, click import items and skills.

### Character build code

```shell
eNrlPWtT20i2n8e_QkXV3i9jQr8l5Sa7Zd7MQCAYksl8SbWkFtYgSx5JhjBb-9_v6ZZsS8YyMpDc2t2ZKiJ1n3O6z6PPo7sF7_7xbRxbdyrLozR5v4XfoC1LJX4aRMnN-63rq8NtZ-sff--9u5DF6DzcnUax7vl776d35tmK1Z2K3285dMsqZHajik8zUvQrkJrIpBipNDmTf6TZURq83_qQJmrL8mQSRMXszY9lnn-QY_V-61ImNyrbsmTuqyTYW3TsKxmoBw08kpn0C5Wd6qEH0yI9SwMACGWcQ_dYRskw9W9VcZSl0wmwtGXdReq-BDo5uzi_vKrNK0rq8wK-fnp3EcsHlQ0LWVg5_Hi_NQDxyBt1HBVASsZToIO2dtbC7k6zvNiXY3h8Gmc4USpYgL0hlPA22ItMHYSh8ovoTu1lUbE3konfYYxNYc-mcRFN4kjrooLHb5w2jONHxDFqJX-VFjLevxg-PZESMu0g9s9RMdqNQYyd6Grok5skKlRn8Is0ytNko1l3At6bxjEsqE6wlypX2Z0soo4T2UvHXpR0lMlemsZBep_MIdkbSmgb9JlM5F6ad9CMhrxQGazmYiOEofJTcACbjrEh5mkUqu6QG_FRIWw6m-fxcTDsCrcx4edN6BI8XzfIYTqNO0IWC2fksvZV8mcDsNWG99W3hbviZA29BiRr9YInyYIJIsQagnVI7DjtM7xL9Wrv6HMOji_mkDa2IZBggV1BHeq2hpPRQx75Mj6T36LxdAx-_EreqsWAFAvSbq03oyIB59WGjIloF9ZhlKlWRETQGkcVB62IFNutoRMiVJq3YtquvW5ZLhTrOmxNIPffauiTxF9QxmvX-3WSGbdeywDWDaFRLmEd6jTDi1VnnMUw1XruEqnLwW5UUo340M2FnSrlj44gD7uUhermvRfLldhrpauB69IlzjqyK4S7jW3RDWVZUNvYeeM63LUFETZfGxpXCw2LN2Qd1oZyO0hUdvMwHEUqrtmOY3eBn81tT066oBrR1_EbBs67jbiRHdVRN5TLZ5kF3QLUhnMyTtaUPfsaseZ10Jv21WewPqjisoGzjTF7I1qRAH6--Go4awcCnLntdbI5QHikoQ6auZN5PSbZwhHr7aZEqJuM0xqQzhRk8YARqKWCoj1AX2TpH7ogijfEG2TjdJp1Ks_8tyVwnQf0VEwty79LFUz9bkF8XtjtxlDALvFBxJNoMNE4XoWLW21mUBTSv91PgxvVtTo0o2yE0ZzgcDqZgIvVJrFEwHHftPtmnS9AARTVEkXhPg18Dku77uTWphVL9NsnswBeps_Qk-nS8ii8I8byUNhdm-ksDbMtUAfwzuKa6_QMnOcYwqTZxDhLg9p2gY3aHdwh1L2d6lID2LGYvkjvYfojvQ-VbwYNaeFCha1TyVTy10Nn-g3wTgMcJME000ui8xjLGPVhZhnKVTQGp5rn-7KQVlCVFJ9kFsmkIGZ_LFcy80enoP9DGcceeIT3W_VW_baEiDWipv9ux0RE_XQynqRZYcUyLwa-n06T4ljmI5gIIT5XGHFJUUBsj0oqUOi6oUNsyQPFXOTj0KNbBvVSyXgM2tkrX_dm230lLSUCFCBMAoGVcEPmUAcp7lLEMRQbrudiroQX2FuW-qZncyGz4mG2OWjma1qAnbyIErOXAl4wjres4Si9HwR3WphXaRrn8x1FOZmoJGjQuMqUsuTMp_laFkbU-sUaw6whnyhXCJD5J3Opi_uCcYz_1f8nY1BU9ZktGIE3yhF1-oxwhuCNCCgv-67NObzYzHH6ghBu4Ijgdh84dDQcpw6BPkrMGwWKdh8qGEORuQ5Bfcah7V-NfdQT4IFsWUkKvMMTh-n0KcGAyoVgol9OhdiO7fYxFo7bJxwJIOwi5PQpzATgocLifY4cAkiYsj6njAvd6dh6PhzDHB0bOGIO7nPuOKgvsOC0TynFqE-EpkiYw_RINjDICaKA5ArE-4IzAr3Y5TAed4AYPCL4yQQQEw6MJDgBMiA7CuDUZTADYmMGz4zBDIgNLTbwjyFTd_oOAmwGgmR97NjC7QvEGMwCOQi4dh2qf8KU4CcBpFKQLtENxBV9KKpsBNwy7va5owemAhM9FeJyUBoQNEoC1piWBwMwoAaIDFOqW-ARuQykgWEKIB2QDGeYkT78sKHTFsBTaRGgMKACNKjou4iCBgkiIHAQLCdaXhQUC7IQhIE0idDyBQxAd0FBMFfGYSCjfBjc1vMDwlr6IFMmXATEYV6kXxoZzAgmih2sVcmQpunaWI_CNYeYE9anjqv5pFplMG1tLswoS9szDAEkKXIJzJfaMGEYCABtrBXJYe6gUpf0XRgb0EHzWhbczB1MuTRh4M7WUgVlaYej95dk9jBoGm0SwfIsYMnVzhYIr44NSqvWq--nd9eXp-bhp1FRTPK3Ozv39_dvJrIYpaH6BvnhGz8d70wACdbtdn4bxfG2JrszgP92b072PHKehL_9FlxCQumd-8Wfn4-z4e3eh_C3ydWBm3mXxf7HP7yv9MC2pwdfjuXwYv_TzYjHiBx89E7-Or1wjg-_8g-fvqX3p1d76LAIhdie3vGHM_onuDt8NDj9ePX7SfjRi53hp-P86PDwWHg0xdPgenSD_nBHw_vgV_uPLyn6nWz7H7NLPNi_OPU-Bfneb7fXn5zBzTT7ZRR9Lj5spzeX-d6vyP-LpuLz4Pw4_-XkzHf4_Zihrz7dnnxzsj37-Oz8fnKPLsRvp87plJ7v7zq_R8M_olv_CH2ber9MJqOH6-GX628HF1_Qh-3M2b-7TXfZX9EwKM6ufx1__ars08Ht18-_5tdB_PU2-OVg-OnXi-2bwS-T6OGE3n75PLqhf53u4sF09GdxfHpwCOONj8iX2-2vXwe27V6H_JieffjlwChoZ6ahd-URTb5Tvum0I4vAL5URa0f7URNbtKPVDx_SQuW6TzfOXt4NtQpzKwc_f6TG-e4DZAmHupRZ2vuuPLWGHqqiDHl1nPdbRTZVOsaFchrr9o9TGUc6cKB662l52JWk2Xi-ZQakIHDopLakePUw0e51cHpa9gzioiKmh5tFkTJaVBOyomAWQapGc4o1WMx6T8Z-buYdJX48DdRJUiVQ1eqIpadnps_u9LZQMOMoj1Pg9yiGJC3fWqY6H_SndzC3ChVgPRmTGd2SYwyr80aN9Vo7U4UMIJPYOSmAnx3N1I6hBk-DzJeJ2otTeauzEJMoaJxGu1l3j1r_rAm8etYw-6XstxqTwzPulpg1KUc18USOq9BcDmOV4-y8Lr_TIh3LWXG3YLfWvOC21vgdmV2M0pVVtp7VuWFD7ZspmcOSgozzEc-PeivEmgSqlhV0vp88ztJMWfOBOoqEdVL-YQyj7ENS2pBDrXXOeK3t-3FqBrHMKJVTLWf6tFcxhJ7lVz4rOUkTC6_wLBq7owHancQ9jG6i-Dw0RVtD4s2OudD3osyP1bz9-wnejG-loVWO9Loe5lhlMg7Ow5Nym2TOdKN9znOj9ftxXA6jWdYD_ZtZWze578ZpGlxWtzfmUq-1zmVea_t-EjeDWGaUHy5ua3ivN4ReIvOOEWYQBJH20zL-7aIZUGsdrXGlif39dHGQxHpfTiU_ThW7aVrkP8LsZw7kajRNgiU_-6jvkduZ9_wI1zMbrKu_dTsJ4CJTfpQvJze11kU-5_uQVvgPAyhUs6jIB_D2PTlfzOF1I8wRMNH0clXLnNPq_fuxVg7wox0bedmC6mpPs7MpGPU2UXm-ZFiPuxd-LYkmKsvPZPZd66NymP8BmeT_m1tmtFeysLlj10TPk-r-5Jz5eWurS6_hfccSAQaxzhNLD_PIBqtHKNLNdkS54aAfjRUaiJNkMi0Mvfdb4yj3v3rTMNS3TYGfIjPXaA8ODw_2rk4-HVQ78nUUw_PXZDr2tOcs_53trcMKUObs0MqnXl4-vt_6FKl7M5F9EHoU55qxOJaTXM03qc2OQjXzGPDWUDNQwPvssupqWguAdkoH31RWAL-fZeZnkWqd17z_iUmVA-qzeX2C0UZNXw9tJ1Seb-7pYwVzd6BFUuYCbjsV4-DbkHXnGtyT8UTGrSNXvU9IotB7SmCsURj5-vBnvcr1DlQJtUYuVfRao-_qcLadhrlv20ag7GxHLu_TtmFXvWukam7vtkq17G1H31e-bOW97GxHnl9HSUvXtJrKHGoNpQ9pYowcFs0givUpaqtmD2I1B2kneF6MVFYdPLVROgMfNQNZu3CyyJsW7cu4BrFGVubqXIuEdF87anktrIUH3bfGEzVuRrUItA7TTqq8TNTqyNahlifqrfKrzufXqKC6n9Ii_rJ3jRBm93Ra-K-61ywS438Hd2kUlNc0WpbLEtg6h5H6ty8nY-6evJzM8mWUl1M8jGV-26rvqrcd_RryQEhrVlAp05dORPSiehkFvbZeRuEK0vJimqlnE7hczkQWuJfrc5D5xYmVyLPedZ6juk_xbArlrY9no1fblc_ENv4fcmIFHKwNAHOYNeujmCaLLfoXkTLTWu1IFtxtRKsMhSs53ZhiucCr6-DrfEAJ8gQhiOXHa7LFbpTmt6uOlYz1x0dp_DKCj669v4SYvjE7ncgkmJE7X5WjL_TQUXppkQPNxf3al8owUeOHFYTa5_VuZ1bVmQs-us6qLkENi0xvJ_yVpuMvpgTVT9VpMK1OgCHj3o9Ab5mxt9k4GvC32QWud6Y6ro6j9fP8NLqdwDRX5Xc35baFwZhtUxh69dPjS6iui4e31vWHk4_XB739VAWZmpX2VyoBp9z7pOI7VVjlmXCvTIOsMg96a2HSqydGuzKffZAUxeqthd4IIezedRL9OVXWyf5bizs2cQJlY1eEbijcILRtFQpKoFX6NlUOJ1QygQgSIbFJSBAJZBj4ruTKt3HPsGBE-day7V51Vg5D9aorAm-t3e3d7cvt3Z6BulR_mmlC5RRHfqT7Ue9nRqwitU6SAmJ6dKMF2MMI_U1vEJWHjZYJ91ap8h7mf7MyfTEXOnRxaJkK0NT-Z3o3P7lRVqZ_6s8x-VYl4mp3oBWCPAkxu5JntkmaGiRNDV4OLg96h6mCmUWZjHtX9-n2sEgTZV2Ch6lrwA-kLYIAxOkppnDoC-UKxwsEp8qTdkBcRF14obYLz1TSgGP4D8lAcdtTTkMDDq9JmbO6lHHvZ0z-psWsb79asFSt-QVVq0xmtXfPez9zV0ONS5dj6XwC2gymBBVAJaN9t4ybSJQaCH1zt9YB7Y5pXzVU759-JsNCBf_6meP6kDoJ-mHKXAPBnoTgT0KIdpOhKxf9RRbl43sl71TvcpoXkW8NZT6qWwwLAxa6ykXMlx52qfDgwVeIhr5PhXSF9H1GQh8HtgRjgkUrfRJizD1fr3PSXLOsZjGENy2GsPoSLHfnrNn9-NliHARBbmGmFUhJaQBlj24pN3DyCgjpJsJKC2wD0i2C1SzmMeDP-Cl7pA3nsYCpaN1HxaiiZlW3kCBx1IPJ5MEyCa5VBvH_AjtkK1zXBfgHNU6TCJbj0QOEnXYHRpVwEMbKQY4gIQpsJ8QBVgIxLAgNPJv5ShFuBwFjnGCbCywDN3R9Bu6LOXbTgdXNEULVkgOrORntwIwdNdyQY0xstj1mXUq9ownuizzyLz8zsdpjsZqXrLX_O9nBKi3zFVr-NdPJk3WUyCwGUXtTWCufQbCNDEF4rk0RRKjA5W6gr40GDkb6trZ0OfJoaEsiHN9VnmNz2xUBIRJMgEH8QoEky-qdZwikkSIcWUf10OU2NU9Fey6AWGtf6VCwMG4HNZxT2ceMt3H4skcyVHRUc1fawsx2Vka1_3yHIZqmdDY4OtnrDf1RGsv5QdQvkJ9UjjQNLSj6rIHO7tKJavgOP8CexwJkUw_5KAgJCR3MIYRRP_RDSEwDQT3sOI5k2EEICVcqH1wHp56HRdN3MFJPP2tRzW5mm9iuZY9lTW5NVKZT9R5zG3Gj3GSr3MgsRHQJDC_KJe2mfD-cX54NTnv70V0E7lfnY6Vg64KEdekz_VkriAgkZIcQ-ZniIDHqMmozZYP0AvC8gtjcYxwcL_cwBg-scOAR1hCkwKsFKVBTkKtn76y0DqWKkRbeMcTs9F4FxgkvDOQihvj8UOdIuDwkLpMSSxe0j4j07MALA64cQr0gcIgfhjYjjs-lkMz3he-7YeiEQeBxX8oGRxyt5ogtFSJCLExjMNbnjVb1fSmUFyeJXuHzlode9QChyJLz2yuWNi_gyLA2Ww2G2RnejLLBhZyovDefW1GoTTCw7kdQrFlJChCJpU_Gf2xCvEqnbocVP4xizVBjzQ9vp0nDTl2MKHXdANa3_vghsBFHijLIUR2PcwoG6xPBecAlc6EQxQwSCqhAccAgw4UfzQWPOi54Z82CB23pr6vmt0wBdTxNgKZ28OVB1ndZ-E-771WKwGiVJg6SwPDwcRpBerxCD3vZA9hu3HC92rkqJChywUMQ33E9KPelsEMoLQl3GKWODERIkUeI4qENqZstJQsCLmTgi6Ym7NWaoKypCWLX3Wupi8V6whDJawtpZWb_w5wwxisSpgN5E5c1vert6_x4ZvzHaZY07NyB0EY8HAYSOzZhjIjAYSBuFTqQCRMHCQRJlfIJwkpnyA52AzuEdBmyLT9Aa_zxImmy6nsqXDRTJpPfUK6tGAr6pbKszG9MDj2rqfTmQEsa1SwDaxsu6I0u-7X7ntWEVU1VjWQOUQFF5rOcG_FV-fm_e26NV-3-XEZhYV2A2-oN8lx_tDQ3FnMfsFclF7BIbDF7ebxh5xCGl3f5OFu7y4frhqgkFbrYEpBEKVc6wkMOdgLs-w5XgY2JIjZxbCG4jagMwT4RCoVLmHI8Fuo8vG6ItDV91__XUwW6FFjRqrxK21-Dt_k-0qOMm-D2jJs4deKzL5orG8W4vVOH3Id0ao3knaq0U1iQEpnbRNovAWT8UNunEqa8bKaF_wUJP6Yr7HsvzbLpxOzb7yoZ5L2BvvsEyc00VkXdBEkouCupRKHrmT0CSOcJs6GG5LZDFeO-DymAx7CrfCgasbTDwBUuFq4nfek0Y77TnpBisBKjn2GRqeSmGGl_82hfmdBWP8cbbm5REeqdU_FoA6ERyiq_ZzYlfvyG9P-jZbCVm5i_KxXPvN1AayiMVNa7SOMoHxmpR7f6n3LNzx0hprjNET7ygRjx7k4QhZhwz1U2CTD4Ow8xvT3hYN-BjIZ71GeEMiGhmvBJSKVAtitDCeUHFhDBfNp0gqtPOY6spgdkS1sYegvdN3eyjIlWFxeWDLD9yONn26lb4JLXFPYj82y4vQH8q-N0mTWZ5Necr1l67xP6s8rV9a3pxGQM6G-9L5VjtH5XMk6LmTrPZTFq95sj8JszYv8N5s9Xmj8Ehpm4PqgcnGGW69L9QsXFwtptm7aHfWFDUlDf87e5dJBLHQ4-E4Gtgpf0IKp7RP8yBVcy24YSX3kiFGGIoOIHj-V7kgjqMSZV04cK1G7BjXM6zpcKKb7C55WuziYrjgr03UqTFoJHhhVvlXcSgQxq32XpkcYgpWTNx0rm8DWE0hmCOloX8VGz3DO_fgLK07_-w_fm9KfIbcfNzW-UYTlbEWBceNeXp_pkvLzLPPvOWHed1AXShrD4pGuGwp9CMbmvNfAeICcGuzC2Z_EFAfQkAVhEy_hkA_yyLqY1LlFXNs2nVMtjsw3GnhN6Af-zzx9egcZqhsQrMCRegaFn0VjFEN7EulaZJ30FiZDX4mYTeztW8VgVL5LHitUmXjwDsjGF2nLlzxuTvpYC-CtYA36F9YFfYR5sQ1NYgNMX-Rjy0iXJXkGAm0eNmurtF_FPX8o_eeEa3mwCwYNV3ld_iQ2W2cVLKKxe2PzFFMRrhU78Wj5mc9OsDe08Twrsxbrkr2NUmzBf7jbVogN9riukr-BR6AtXJXshPn4dBTw7VG5stTWG3W4YNeMQXedWMw_2HLHyl3rrV9ILey338mqpNX2u56yp5MkqU184qyuRPM87kddRwqsFi9co-tjzeNoAywi_Zr_secJ_pXyDvzhGvTzKiU29Qc3WybPcx-MRy52f2WftZmfMfNWeJmF0U32gUL5UnygYpHlLYzfoIpa-GqVxoLJqEkpvDVd_1Wj2RbuN5sy2INT_RNEMiz-B0_xd27XR2vGGIxDK0JxZP_refiW8n5ovQDojmIlVv_w7jWOzA1eXwxOYsxU2H4m0I-S1330FMzRfvnWY4Xj-x530nzbSd5MMe3v6VtFQxWGNyFMKmH9lOsMQdgc1L6uLzs1ybmLl9zjm7e-9dzvLf3_s_wAlWuPK
```

**Comment by manweCZ** [comment]
Same happens for me for new bases

Item Class: Helmets
Rarity: Rare
Entropy Glance
General's Helmet
--------
Quality: +20% (augmented)
Armour: 860 (augmented)
--------
Requirements:
Level: 73
Str: 170
Dex: 66
--------
Sockets: R-G-R-R 
--------
Item Level: 79
--------
+16 to Strength
+268 to Accuracy Rating
60% increased Armour
+45% to Fire Resistance
12% increased Stun and Block Recovery
+66 to maximum Life (crafted)


wont get imported, neither will 

Item Class: Body Armours
Rarity: Rare
Fate Ward
Titan Plate
--------
Quality: +20% (augmented)
Armour: 1327 (augmented)
--------
Requirements:
Level: 73
Str: 222
Dex: 95
Int: 99
--------
Sockets: B-R-R-R-R-G 
--------
Item Level: 76
--------
18% increased Armour (implicit)
17% increased Global Physical Damage (implicit)
--------
+49 to Strength
+76 to Armour
16% increased Armour
+189 to maximum Life
+23% to Chaos Resistance
8% increased Stun and Block Recovery
+32% to Fire Resistance (crafted)
Searing Exarch Item
Eater of Worlds Item


When trying to add as Create Custom it says

"The item is invalid, check the item's title ..." https://i.imgur.com/c05cSO7.png

**Comment by Skuamato** [comment]
Yep, encountered the same issue. Checking in the database it seems like all of the new bases across the board are missing.

**Comment by matheussampaio** [comment]
A quick fix is to edit the base name to one of the old names.

**Comment by zao** [comment]
New bases seem to have been merged in to `dev` with https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/7970, ought to work in the next release.

**Comment by LocalIdentity** [comment]
Fixed in the latest update (2.47.0)
